cmd/clusterctl/client/config/cert_manager_client.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	// CertManagerConfigKey defines the name of the top level config key for cert-manager configuration.
+	CertManagerConfigKey = "cert-manager"
+
+	// CertManagerDefaultVersion defines the default cert-manager version to be used by clusterctl.
+	CertManagerDefaultVersion = "v1.1.0"
+
+	// CertManagerDefaultURL defines the default cert-manager repository url to be used by clusterctl.
+	CertManagerDefaultURL = "https://github.com/jetstack/cert-manager/releases/latest/cert-manager.yaml"
```

Comment:
This would fetch the latest version all the time?

Reply:
No, we are going to fetch CertManagerDefaultVersion or the version the user specifies in the config file.

Reply:
Could we add some text on top of this and the version to that effect?

Reply:
Done!